docs/faq.rst

```diff
@@ -193,28 +184,80 @@ In case you have a general question that is not answered here, consider submitti
     documentation of your package creator (or the software you use to package
     up your Python projects) and the `PEP 420`_ for more information.
 
-|
 
-11. **Can I modify** ``requires`` **despite the warning in** ``pyproject.toml`` **to avoid doing that?**
+pyproject.toml
+--------------
+
+- **Can I modify** ``requires`` **despite the warning in** ``pyproject.toml`` **to avoid doing that?**
 
     You can definitely modify ``pyproject.toml``, but it is good to understand how PyScaffold uses it.
     If you are just adding a new build dependency (e.g. `Cython`_), there is nothing to worry.
     However, if you are trying to remove or change the version of a dependency PyScaffold included there,
     PyScaffold will overwrite that change if you ever run ``putup --update`` in the same project
     (in those cases ``git diff`` is your friend, and you should be able to manually reconcile the dependencies).
 
-|
-
-12. **What should I do if I am not using** ``pyproject.toml`` **?**
+- **What should I do if I am not using** ``pyproject.toml`` **or if it is causing me problems?**
 
-    If you prefer to have legacy builds, you can remove the ``pyproject.toml`` file and run
+    If you prefer to have legacy builds and get the old behavior, you can remove the ``pyproject.toml`` file and run
     ``python setup.py bdist_wheel``, but we advise to install the build requirements (as the ones specified in the
     ``requires`` field of ``pyproject.toml``) in an `isolated environment`_ and use it to run the ``setup.py`` commands
     (`tox`_ can be really useful for that). Alternatively you can use the ``setup_requires`` field in `setup.cfg`_,
-    however, this method is discouraged and might be invalid in the future. To skip the generation of the
-    ``pyproject.toml`` file you can run ``putup`` with the ``--no-pyproject`` flag.
+    however, this method is discouraged and might be invalid in the future.
+
+    .. warning::
+       For the time being you can use the **transitional** ``--no-pyproject``
```

Comment:
I was just thinking. Should a kind of [not recommended] statement also be added to the cli help of `--no-pyproject`?

Reply:
That is a good call! I will merge the PR and then add it to master.